docs/databases/postgresql/how-to-deploy-ha-postgresql-cluster.md

```diff
@@ -0,0 +1,406 @@
+---
+author:
+  name: Kulshekhar Kabra
+  email: docs@linode.com
+description: 'Deploy PostgreSQL in a highly available setup'
+keywords: 'postgresql,clusters,databases'
+license: '[CC BY-ND 4.0](https://creativecommons.org/licenses/by-nd/4.0)'
+modified: Wednesday, August 16th, 2017
+modified_by:
+  name: Kulshekhar Kabra
+published: 'Thursday, August 31st, 2017'
+title: Deploying a Highly Available PostgreSQL Cluster
+external_resources:
+ - '[PostgreSQL Documentation](https://www.postgresql.org/docs/)'
+ - '[Patroni Repository](https://github.com/zalando/patroni)'
+ - '[Etcd Documentation](https://coreos.com/etcd/docs/latest/)'
+---
+
+## Introduction
+
+[PostgreSQL](https://www.postgresql.org) (Postgres) is an open source, fully ACID compliant relational database that runs on all the major operating systems. While Postgres is a highly versatile, feature-rich and powerful database, it doesn't have a built-in solution for high availability.
+
+This guide will explain how you can set up a highly available Postgres cluster of three servers using [Patroni](https://github.com/zalando/patroni).
+
+## Before You Begin
+
+1. This guide assumes that you are familiar with setting up new Linodes with private IP addresses.
```

Comment:
This just involves hitting a button on the Linode Manager. I'll will add steps for the sake of the user.